helm_sdk/__init__.py

```diff
@@ -138,14 +139,20 @@ def install(self,
         server.
         :return output of install command.
         """
+        flags = flags or []
+        validate_no_collisions_between_params_and_flags(flags)
+
+        for item in flags:
+            if 'repo' == item['name']:
+                if '/' in chart:
+                    chart = '/'.join(chart.split('/')[1:])
```

Comment:
```suggestion
                    chart = '/'.join(chart.split('/')[1:])
                    break
```
Seems like we don't need to continue looping after we found this.